code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -153,31 +153,31 @@ GLOBAL_LIST_EMPTY(PDAs)
 				dat += "<li><a href='byond://?src=\ref[src];choice=2'><img src=pda_mail.png> Messenger</a></li>"
 
 				if (cartridge)
-					if (cartridge.access_clown)
+					if (cartridge.access&CART_CLOWN)

```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#operators-and-spaces